src/app/Sidebar/UserInfo.js

```diff
@@ -75,6 +75,7 @@ const UserInfo = ({ intl, authenticated, authenticatedUser, user, ...props }) =>
         defaultMessage: 'Transfer',
       })}
       onClick={() => props.openTransfer(user.name)}
```

Comment:
Can you add handler for `onClick`? This creates new function ever time `UserInfo` re-renders.
We usually put boolean type props first (`primary`).